### ISSUE TITLE
Installing on centos7 with agent 2.2 fails

### ISSUE THREAD

**Issue by rkarlsba** [cavaliercoder/libzbxpgsql#89]
Seems the dependencies are messed up with 2.2 - it checks, but fails to see that 2.2.16-1 > 2.2.0

[root@pgc1-dev ~]# yum install libzbxpgsql-1.1.0-1.el7.x86_64.rpm
Loaded plugins: etckeeper, fastestmirror, priorities
Examining libzbxpgsql-1.1.0-1.el7.x86_64.rpm: libzbxpgsql-1.1.0-1.x86_64
Marking libzbxpgsql-1.1.0-1.el7.x86_64.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package libzbxpgsql.x86_64 0:1.1.0-1 will be installed
--> Processing Dependency: zabbix-agent < 3.0.0 for package: libzbxpgsql-1.1.0-1.x86_64
Loading mirror speeds from cached hostfile
 * base: ftp.uninett.no
 * epel: ftp.uninett.no
 * extras: ftp.uninett.no
 * updates: ftp.uninett.no
--> Processing Dependency: zabbix-agent >= 2.2.0 for package: libzbxpgsql-1.1.0-1.x86_64
--> Processing Dependency: libconfig.so.9()(64bit) for package: libzbxpgsql-1.1.0-1.x86_64
--> Running transaction check
---> Package libconfig.x86_64 0:1.4.9-5.el7 will be installed
---> Package libzbxpgsql.x86_64 0:1.1.0-1 will be installed
--> Processing Dependency: zabbix-agent < 3.0.0 for package: libzbxpgsql-1.1.0-1.x86_64
--> Processing Dependency: zabbix-agent >= 2.2.0 for package: libzbxpgsql-1.1.0-1.x86_64
--> Finished Dependency Resolution
Error: Package: libzbxpgsql-1.1.0-1.x86_64 (/libzbxpgsql-1.1.0-1.el7.x86_64)
           Requires: zabbix-agent >= 2.2.0
Error: Package: libzbxpgsql-1.1.0-1.x86_64 (/libzbxpgsql-1.1.0-1.el7.x86_64)
           Requires: zabbix-agent < 3.0.0
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[root@pgc1-dev ~]# rpm -qa|grep zabb
zabbix22-agent-2.2.16-1.el7.x86_64
zabbix22-2.2.16-1.el7.x86_64
[root@pgc1-dev ~]#

**Comment by cavaliercoder** [comment]
This is a duplicate of #18.

You have three options:
- use the package from http://repo.zabbix.com/ instead of EPEL
- repackage the rpm package from srpm and fix the `Requires` directive
- compile and install from source

This issue sites with the EPEL package maintainers to fix. See https://bugzilla.redhat.com/show_bug.cgi?id=1269204
